BWEnv/src/controller.cc

```diff
@@ -387,16 +387,19 @@ void Controller::handleCommand(int command, const std::vector<int>& args,
     case Commands::DRAW_UNIT_LINE:
     case Commands::DRAW_UNIT_POS_LINE:
     case Commands::DRAW_CIRCLE:
-    case Commands::DRAW_UNIT_CIRCLE: {
+    case Commands::DRAW_UNIT_CIRCLE:
+    case Commands::DRAW_TEXT:
+    case Commands::DRAW_TEXT_SCREEN: {
       static std::unordered_map<int, int> argcount = {
         {Commands::DRAW_LINE, 5}, {Commands::DRAW_UNIT_LINE, 3},
         {Commands::DRAW_UNIT_POS_LINE, 4}, {Commands::DRAW_CIRCLE, 4},
-        {Commands::DRAW_UNIT_CIRCLE, 3},
+        {Commands::DRAW_UNIT_CIRCLE, 3}, {Commands::DRAW_TEXT, 2},
```

Comment:
The third one is the text, which does not count towards the argument count I think?